lib/src/widgets/chat_list_widget.dart

```diff
@@ -184,6 +184,7 @@ class _ChatListWidgetState extends State<ChatListWidget>
             valueListenable: showPopUp,
             builder: (_, showPopupValue, child) {
               return Stack(
+                clipBehavior: Clip.none,
```

Comment:
Please see if this is absolutely required.

Reply:
Removed. There are more of those, I didn't check the ones I did not add.